pkg/controller/container-runtime-config/helpers.go

```diff
@@ -172,10 +176,54 @@ func getManagedKeyCtrCfg(pool *mcfgv1.MachineConfigPool) string {
 	return fmt.Sprintf("99-%s-%s-containerruntime", pool.Name, pool.ObjectMeta.UID)
 }
 
+func getStableManagedKeyCtrCfg(pool *mcfgv1.MachineConfigPool, client mcfgclientset.Interface) (string, error) {
+	// if we don't have a client, we're installing brand new, and we don't need to adjust for backward compatibility
```

Comment:
"installing brand new" means "on the bootstrap node"?

Reply:
yeah, fresh installation